package.json

```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "clean": "rimraf commonjs es coverage",
     "lint": "eslint --rule \"no-restricted-globals: 0\" test && eslint --rule \"no-restricted-globals: 0\" --ext .unit.js --ext .e2e.js src/ && eslint --ignore-pattern \"src/plugins/**/test/**\" src/",
```

Comment:
Could be changed according to comment:  https://github.com/handsontable/handsontable/pull/4724#pullrequestreview-89749164

Reply:
This line was introduced while fixing linter for Windows. After changing this line it should be tested if everything works.